hnn_core/network.py

```diff
@@ -94,7 +94,7 @@ def _create_cell_coords(n_pyr_x, n_pyr_y, zdiff=1307.4):
     return pos_dict
 
 
-def _connection_probability(conn, probability, seed=0):
+def _connection_probability(conn, probability, conn_seed=3):
```

Comment:
Out of curiosity, why did you need to change `conn_seed` to 3?

Reply:
Honestly I don't remember :smile:, perhaps it's a good reminder that the seed can be any int.